_includes/live-demo.html

```diff
@@ -43,62 +43,62 @@
   {% endif %}
 {% endif %}
 
-{% comment %} Assign html to codepen_html {% endcomment %}
-{% capture codepen_html %}
-{% include codepens/{{ include.id }}/index.html %}
+{% comment %} Assign html to live-demo_html {% endcomment %}
```

Comment:
More inconsistent naming here, this should probably be `live_demo_html`.

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/1645/commits/8bc36cb9e57a4c890e01fa0a70ebc3484a22a7bb